03-Custom-Actions-In-Python/04-Custom-Updater.ipynb

```diff
@@ -0,0 +1,386 @@
+{
```

Comment:
<p>What are your thoughts on splitting the "non-trivial" stuff into a new tutorial page? It's weird having to remove the previously-created updater.</p><p><code>sim.operations.updaters.remove(energy_inserter)</code></p><p> </p><p>I see it could also be confusing to have duplicate "boilerplate" stuff between both.</p><p> </p><p>At least I would suggest adding a large heading to the second section.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>I missed the very subtle change of </p><p><code>self.energy</code> --&gt; <code>self.energy(timestep)</code> </p><p> </p><p>We should call attention to this somehow. Simplest way could be with a comment on that line or by adding a bolded sentence in the previous paragraph</p>